components/sessions/SessionGridCard.tsx

```diff
@@ -21,7 +21,71 @@ export const sessionGrid = () => {
         </div>
         <div className="w-full lg:w-11/12">
           <div className="px-0 md:px-6">
-            <p>Cards hapa</p>
+            <div className="lg:grid gap-4 grid-cols-3">
```

Comment:
@i-Krypt i think what you need to build it this card alone, the upper shell already exists